.gitignore

```diff
@@ -32,3 +32,5 @@ resotocore/tools/Assets
 out/
 *.iml
 **/.hypothesis/
+plugins/aws/test/resources/*.ipynb
```

Comment:
I guess this is might be helpful not only in the context of AWS - maybe we use a directory on project root level?